cdap-docs/integrations/source/partners/cloudera/ingesting.rst

```diff
@@ -99,59 +99,70 @@ are written in a custom format, they cannot be directly queried through Impala.
 you can create an ETL batch application that regularly reads
 stream events and writes those events into files on HDFS that can then be queried by Impala.
 
-To do this, write the following JSON to a config file::
+To do this, write the following JSON to a config file:
 
+.. parsed-literal::
+  :class: copyable copyable-text
+           
   {
+    "name": "streamToTPFS",
```

Comment:
name is added by the UI and is not used by the backend. The application name comes from the REST call and nowhere else.